src/main/scala/strawman/collection/immutable/HashMap.scala

```diff
@@ -28,8 +30,9 @@ import java.lang.{Integer, String, System}
 @SerialVersionUID(2L)
 sealed trait HashMap[K, +V]
   extends Map[K, V]
-     with MapOps[K, V, HashMap, HashMap[K, V]]
-     with Serializable {
+    with MapOps[K, V, HashMap, HashMap[K, V]]
+    with Buildable[(K, V), HashMap[K, V]]
```

Comment:
I think that the decision of extending `Buildable` can only be made at the leaf level (for each concrete collection type). For instance, `Seq` does not imply `Buildable`, which is good because we don’t want `LazyList` to implement `Buildable`.